### DISCUSSION TITLE
Soft deletes

### DISCUSSION THREAD

**Discussion by shivamvijaywargi** [GenieWizards/zenmo#60] [Ideas]
As seen today, should we do a soft or hard delete?
@GenieWizards/work-o-holics

**Comment by sub1120** [comment]
I think from user end we can give option to soft delete and admin can do hard delete also. In case of hard delete admin will be asked to confirm the message "This will permanently delete the group and its associated split data." + we will record data under activity metadata.

**Reply by shivamvijaywargi** [reply]
Agreed, this should be good enough for now.

Later what we can do is only let the admin delete permanently if the user has given his/her consent.
